contrib/images/conf/postgresql.conf

```diff
@@ -0,0 +1,780 @@
+# -----------------------------
```

Comment:
It is likely better to append to the `postgresql.conf` or do an `include` with just the changes that we need instead of maintaining complete files across all versions